app/src/main/java/net/skyscanner/backpack/demo/data/ComponentRegistry.kt

```diff
@@ -291,6 +292,12 @@ object ComponentRegistry {
         TAB_TITLE_COMPOSE composeStory { BpkFabStory() },
       )
     ),
+    "Floating Notification" story NodeData(
```

Comment:
This looks similar to BpkSnackbar – what do you think about adding it there?

Reply:
no this is separate to backbar from the figma side - we should eventually deprecate snackbar instead